### PR TITLE
Added function to detect expansion pak

### PR DESCRIPTION
**Pull request by joeldipops**
Just a minor and simple thing I thought was missing.

### CODE REVIEW

File: src/n64sys.c [hunk 1]
```diff
@@ -222,4 +222,17 @@ void inst_cache_index_invalidate(volatile void * addr, unsigned long length)
...
+/**
+ * @brief Is expansion pak in use.
+ *
+ * Checks whether the maximum available memory has been expanded to 8MB
+ *
+ * @return true if expansion pak detected, false otherwise.
+ */
+bool is_memory_expanded()
+{
+    int memsize = *(int*)(0x80000318);
```
[the remark this answers is not in the document]
Comment (joeldipops) [reply src/n64sys.c]: Yep makes sense.  What would say is the difference between n64sys.c and system.c?

[the remark this answers is not in the document]
Comment (anacierdem) [reply src/n64sys.c]: `system.c` is the compatibility layer for newlib and provides low-level C API. `n64sys.c` contains n64 specific functions (its brief says `N64 bootup and cache interfaces`). I think it is ok to put is_memory_expanded to `n64sys.c` and use it in `system.c`.

[the remark this answers is not in the document]
Comment (joeldipops) [reply src/n64sys.c]: Have updated the branch.  How's about this?

[the remark this answers is not in the document]
Comment (anacierdem) [reply src/n64sys.c]: LGTM, will test a few examples and merge this one...

[the remark this answers is not in the document]
Comment (joeldipops) [reply src/n64sys.c]: Thanks

**Pull request merged**